releasenotes/notes/cleanup-timeline-drawer-a6287bdab4459e6e.yaml

```diff
@@ -0,0 +1,27 @@
+---
+features:
+  - |
+    New attribute :attr:`op_start_times` has been added to :class:`~QuantumCircuit`.
+    This information is populated when one of scheduling analysis passes is run on the circuit.
+    It can be used to obtain circuit instruction with instruction time, for example:
+
+    .. code-block:: python
+
+      from qiskit import QuantumCircuit, transpile
+      from qiskit.test.mock import FakeMontreal
+
+      backend = FakeMontreal()
+
+      qc = QuantumCircuit(2)
+      qc.h(0)
+      qc.cx(0, 1)
+
+      qct = transpile(
+          qc, backend, initial_layout=[0, 1], coupling_map=[[0, 1]], scheduling_method="alap"
+      )
+      scheduled_insts = list(zip(qct.op_start_times, qct.data))
+
```

Comment:
Did you want to add a deprecation note about visualizing a timeline for an unscheduled/untranspiled circuit?

Reply:
Thanks Matthew I've forgotten to write. Added in https://github.com/Qiskit/qiskit-terra/pull/7935/commits/8891220d691110471dc5bb145b03e6487cc81b82